docs/services.md

```diff
@@ -1,114 +1,185 @@
-[Quay](https://quay.io/ukhomeofficedigital) is where store all of our open source container images.
+For more information on Drone, see the [Drone documentation][drone_docs].
 
-#### ECR
-We use AWS ECR to host our private container images for high availability and resiliency. For more information on how to use ECR see [here](how-to-docs/drone-how-to.md)
 
-#### Artifactory
-We use [Artifactory](https://artifactory.digital.homeoffice.gov.uk) as our internal binary repository store where projects can push build time artefacts and dependencies. Prior to the introduction of ECR, Docker images were also pushed here and can be accessed via https://docker.digital.homeoffice.gov.uk. Please note that we regularly remove container images that have not been downloaded in a year.
 
-## Domain Name System (DNS) Pattern
-To standardise on how services route their application traffic to the appropriate hosting platform and to offer consistency in how we approach DNS we have a standard DNS naming convention for services.
+## Container Storage
 
-In parallel to this, users need to also be aware of [the limits on certificates and letsencrypt](https://letsencrypt.org/docs/rate-limits/) if they are wanting external TLS certificates for their services.
+ACP recommends separate repositories for open-source and private container images:
 
-If you require a service.gov.uk subdomain, you will need to liaise with the Government Digital Service to get it delegated to our control. Please note that we will not be able to point the apex of such delegated domains to our ingress controllers due to the limitations of CNAME records. You will need to use a subdomain for ingress (e.g. www.foo.service.gov.uk rather than foo.service.gov.uk).
+  * if your code is open-source and hosted on GitHub, use Quay to store container images
+  * if your code is private and hosted on GitLab, use Amazon ECR
 
-#### For Non-production Services
-The following categories are something we would expect a service to specify:
+For more information on Quay, see the [Quay][quay_docs].
 
-`Service Name` - The name of the service users or other services will attempt to consume i.e. `web-portal`
+For more information on ECR, see the [Amazon ECR documentation][ecr _docs].
 
-`Env` - The environment of the service i.e. `Dev`
 
-`Service` - The overall service name or project name i.e. `example-service`
 
-```
-<servicename>.<env>.<service>-notprod.homeoffice.gov.uk
+## Artifact Storage
 
-web-portal.dev.example-service-notprod.homeoffice.gov.uk
-```
+### Artifactory
 
-#### For Production Services
-As we want to protect production services from hitting limits and to create a distinction between services that are non-production, (not prod)  and production, we simplify the overall approach by using the main service name as the domain.
+ACP uses [Artifactory][artifactory] as its internal binary repository store for projects to push build time artifacts and dependencies to, such as jars and python modules.  
```

Comment:
I suggest 'binary store' rather than 'binary repository store'. 
I believe it's 'artefacts' in British English.